SHARE/make_perlmutter.inc

```diff
@@ -0,0 +1,163 @@
+#######################################################################
+#            Define Basic Utilities
+#######################################################################
+  SHELL = /bin/sh
+  PWD1 = `pwd`
+  MYHOME = /global/cfs/cdirs/m4505/sw/STELLOPT/bin
+  PRECOMP:= /lib/cpp -traditional -DLINUX
+  COMPILE = $(FC)
+  COMPILE_FREE = $(FC)
+  LINK    = $(FC) -Bstatic -o
+  LINK_AR = ar -ruv
+  LINK_C  = $(CC) -shared -Wl,-z-defs
+
+#######################################################################
+#            Define Compiler Flags
+#######################################################################
+  FLAGS_R = 
+  FLAGS_D = -g -O0 -fexternal-blas -fcheck=all -fbacktrace -Wextra \
+   -Wtarget-lifetime -fbounds-check -ffpe-trap=zero -finit-real=snan \
+   -ffree-line-length-0 -dynamic -fallow-argument-mismatch 
+  LIBS    = 
+  FLAGS_R += -g -fbacktrace -dynamic -fallow-argument-mismatch -ffree-line-length-0
```

Comment:
Why not just explicitly set `FLAGS_R` at line 17?

Reply:
It makes sense. I will fix to use "FLAGS_R" at line 17.